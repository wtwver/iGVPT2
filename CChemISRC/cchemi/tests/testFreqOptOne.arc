

                     SUMMARY OF  PM3 CALCULATION, Site No: 1079

                                                       MOPAC2016 (Version: 19.338L)
                                                       Wed Mar  4 14:24:08 2020

           Empirical Formula: H2 O  =     3 atoms

 XYZ PM3 CHARGE=0 Singlet

 Mopac file generated by Gabedit


     GEOMETRY OPTIMISED USING EIGENVECTOR FOLLOWING (EF).     
     SCF FIELD WAS ACHIEVED                                   

          HEAT OF FORMATION       =        -53.43270 KCAL/MOL =    -223.56244 KJ/MOL
          TOTAL ENERGY            =       -324.90735 EV
          ELECTRONIC ENERGY       =       -474.27017 EV
          CORE-CORE REPULSION     =        149.36281 EV
          GRADIENT NORM           =          0.65392 = 0.37754 PER ATOM
          DIPOLE                  =          1.73865 DEBYE    POINT GROUP:       C2v 
          NO. OF FILLED LEVELS    =          4
          IONIZATION POTENTIAL    =         12.316182 EV
          HOMO LUMO ENERGIES (EV) =        -12.316  4.058
          MOLECULAR WEIGHT        =         18.0152
          COSMO AREA              =         42.43 SQUARE ANGSTROMS
          COSMO VOLUME            =         25.15 CUBIC ANGSTROMS

          MOLECULAR DIMENSIONS (Angstroms)

            Atom       Atom       Distance
            H     2    H     1     1.53633
            O     3    H     1     0.56094
            H     1    H     2     0.00000
          SCF CALCULATIONS        =         11
          WALL-CLOCK TIME         =          0.160 SECONDS
          COMPUTATION TIME        =          0.024 SECONDS


          FINAL GEOMETRY OBTAINED
 XYZ PM3 CHARGE=0 Singlet

 Mopac file generated by Gabedit
  H     0.00000000 +1   0.77870127 +1  -0.44376571 +1
  H     0.00000000 +1  -0.75754023 +1  -0.46039647 +1
  O    -0.00000000 +1   0.00472840 +1   0.10883062 +1
 
