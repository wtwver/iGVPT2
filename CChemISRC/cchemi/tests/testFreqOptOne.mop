XYZ PM3 CHARGE=0 Singlet

Mopac file generated by Gabedit
H      0.00000000000000 1     0.75695000000000 1    -0.46870600000000 1
H      0.00000000000000 1    -0.71695000000000 1    -0.46870600000000 1
O      0.00000000000000 1     0.00000000000000 1     0.11717600000000 1
