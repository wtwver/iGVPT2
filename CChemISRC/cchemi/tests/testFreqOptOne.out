 *******************************************************************************
 ** Site#: 1079         For non-commercial use only    Version 19.338L 64BITS **
 *******************************************************************************
 ** Cite this program as: MOPAC2016, Version: 19.338L, James J. P. Stewart,   **
 ** Stewart Computational Chemistry, web-site: HTTP://OpenMOPAC.net.          **
 *******************************************************************************
 **                                                                           **
 **                                MOPAC2016                                  **
 **                                                                           **
 *******************************************************************************

                              PM3 CALCULATION RESULTS

 *******************************************************************************
 *  CALCULATION DONE:                                Wed Mar  4 14:24:08 2020  *
 *  PM3        - The PM3 Hamiltonian to be used
 *  SINGLET    - SPIN STATE DEFINED AS A SINGLET
 *
 *
 *
 *                 CHARGE ON SYSTEM = 0
 *
 *
 *
 *  XYZ        - CARTESIAN COORDINATE SYSTEM TO BE USED
 *  T=         - A TIME OF 172800.0 SECONDS REQUESTED
 *  DUMP=N     - RESTART FILE WRITTEN EVERY  7200.000 SECONDS
 *******************************************************************************
XYZ PM3 CHARGE=0 Singlet

 Mopac file generated by Gabedit
   ATOM   CHEMICAL          X               Y               Z
  NUMBER   SYMBOL      (ANGSTROMS)     (ANGSTROMS)     (ANGSTROMS)
 
     1       H          0.00000000  *   0.75695000  *  -0.46870600  *
     2       H          0.00000000  *  -0.71695000  *  -0.46870600  *
     3       O          0.00000000  *   0.00000000  *   0.11717600  *
 


          CARTESIAN COORDINATES 

    NO.       ATOM           X           Y           Z

     1         H          0.0000      0.7570     -0.4687
     2         H          0.0000     -0.7170     -0.4687
     3         O          0.0000      0.0000      0.1172

  H: (PM3): J. J. P. STEWART, J. COMP. CHEM. 10, 209 (1989).
  O: (PM3): J. J. P. STEWART, J. COMP. CHEM. 10, 209 (1989).

           Empirical Formula: H2 O  =     3 atoms



      MOLECULAR POINT GROUP   :   C2v 


 SINGLET STATE CALCULATION


      RHF CALCULATION, NO. OF DOUBLY OCCUPIED LEVELS = 4


          Geometry optimization using EF

          DIAGONAL MATRIX USED AS START HESSIAN

 CYCLE:     1 TIME:   0.004 TIME LEFT:  2.00D  GRAD.:    50.535 HEAT: -52.55455
 CYCLE:     2 TIME:   0.004 TIME LEFT:  2.00D  GRAD.:    51.776 HEAT: -52.74386
 CYCLE:     3 TIME:   0.000 TIME LEFT:  2.00D  GRAD.:    14.652 HEAT: -53.34827
 CYCLE:     4 TIME:   0.004 TIME LEFT:  2.00D  GRAD.:    10.664 HEAT: -53.39518
 CYCLE:     5 TIME:   0.000 TIME LEFT:  2.00D  GRAD.:     1.905 HEAT: -53.43168
 CYCLE:     6 TIME:   0.000 TIME LEFT:  2.00D  GRAD.:     0.654 HEAT: -53.43270

     GRADIENT =  0.65392 IS LESS THAN CUTOFF =  1.00000



 -------------------------------------------------------------------------------
 XYZ PM3 CHARGE=0 Singlet

 Mopac file generated by Gabedit


     GEOMETRY OPTIMISED USING EIGENVECTOR FOLLOWING (EF).     
     SCF FIELD WAS ACHIEVED                                   


                              PM3 CALCULATION
                                                       MOPAC2016 (Version: 19.338L)
                                                       Wed Mar  4 14:24:08 2020




          FINAL HEAT OF FORMATION =        -53.43270 KCAL/MOL =    -223.56244 KJ/MOL


          TOTAL ENERGY            =       -324.90735 EV
          ELECTRONIC ENERGY       =       -474.27017 EV  POINT GROUP:     C2v 
          CORE-CORE REPULSION     =        149.36281 EV
          COSMO AREA              =         42.43 SQUARE ANGSTROMS
          COSMO VOLUME            =         25.15 CUBIC ANGSTROMS

          GRADIENT NORM           =          0.65392 = 0.37754 PER ATOM
          IONIZATION POTENTIAL    =         12.316182 EV
          HOMO LUMO ENERGIES (EV) =        -12.316  4.058
          NO. OF FILLED LEVELS    =          4
          MOLECULAR WEIGHT        =         18.0152

          MOLECULAR DIMENSIONS (Angstroms)

            Atom       Atom       Distance
            H     2    H     1     1.53633
            O     3    H     1     0.56094
            H     1    H     2     0.00000


          SCF CALCULATIONS        =         11
 
          WALL-CLOCK TIME         =          0.160 SECONDS
          COMPUTATION TIME        =          0.024 SECONDS




   ATOM   CHEMICAL          X               Y               Z
  NUMBER   SYMBOL      (ANGSTROMS)     (ANGSTROMS)     (ANGSTROMS)
 
     1       H          0.00000000  *   0.77870127  *  -0.44376571  *
     2       H          0.00000000  *  -0.75754023  *  -0.46039647  *
     3       O         -0.00000000  *   0.00472840  *   0.10883062  *

                             CARTESIAN COORDINATES

   1    H        0.000000000     0.778701268    -0.443765707
   2    H        0.000000000    -0.757540234    -0.460396473
   3    O       -0.000000000     0.004728397     0.108830619


           Empirical Formula: H2 O  =     3 atoms



      MOLECULAR POINT GROUP   :   C2v 


                  EIGENVALUES  
 -36.82134 -17.58048 -14.52173 -12.31618   4.05769   5.33130


              NET ATOMIC CHARGES AND DIPOLE CONTRIBUTIONS

  ATOM NO.   TYPE          CHARGE      No. of ELECS.   s-Pop       p-Pop       
    1          H           0.179322        0.8207     0.82068
    2          H           0.179320        0.8207     0.82068
    3          O          -0.358641        6.3586     1.76343     4.59522
 DIPOLE           X         Y         Z       TOTAL
 POINT-CHG.     0.000     0.010    -0.966     0.966
 HYBRID        -0.000     0.008    -0.772     0.772
 SUM           -0.000     0.019    -1.739     1.739


          ATOMIC ORBITAL ELECTRON POPULATIONS

     Atom    s        px        py        pz   
    1  H   0.82068
    2  H   0.82068
    3  O   1.76343   2.00000   1.17965   1.41557

 **********************
 *                    *
 * JOB ENDED NORMALLY *
 *                    *
 **********************



 TOTAL JOB TIME:             0.18 SECONDS

 == MOPAC DONE ==
